great_expectations/datasource/fluent/interfaces.py

```diff
@@ -227,6 +226,45 @@ def batch_request_options(self) -> tuple[str, ...]:
             """One needs to implement "batch_request_options" on a DataAsset subclass."""
         )
 
+    def build_batch_request(
```

Comment:
These methods are NOT new or modified. I updated the below `add_batch_config` and moved it below the abstract method section that it was mistakenly within.